app/assets/javascripts/analytics.js

```diff
@@ -124,6 +124,13 @@ Blacklight.onLoad(function(){
     });
   });
 
+  // Exhibit link clicked
+  $('.panel-exhibit a').on('click', function(e) {
+    ga('send', 'event', 'Exhibit link', 'clicked', $(e.currentTarget).text().trim(), {
```

Comment:
Would there be any use/desire in distinguishing between a users click to navigate to an exhibit vs. toggling to see more exhibits?

Reply:
Do you have any markup / screenshots of that toggle functionality? I couldn't figure that out exactly.

Reply:
SHPC (Ckey 4769498) is in multiple exhibits in production (and is currently configured to display in stage).

Also happy to ship this, as more work is likely to have to happen to that toggle before we roll this out and addl. GA work can be done there.  Up to you (don't mean to block this PR)

![exhibit-toggle](https://user-images.githubusercontent.com/96776/73491318-364c1300-4363-11ea-8717-78b15ffb00b0.gif)

Reply:
Thanks that got me rolling. I ended up just modifying the approach to be a bit more stable with the async nature of this. No we are only tracking exhibit title clicks which is probably what we want. 🤷‍♂